runtime/drivers/git/git.go

```diff
@@ -18,21 +20,40 @@ func init() {
 
 type driver struct{}
 
+type DSN struct {
+	URL    string
+	branch string
+}
```

Comment:
1. The `branch` property needs to be public (i.e. capital letter), else JSON unmarshal won't put a value in it
2. Add struct tags for JSON to make the field names clear, i.e. `json:"url"`, etc.